src/components/AnalyzedArtifactsList/AnalyzedArtifactsList.tsx

```diff
@@ -0,0 +1,327 @@
+import { ServiceContainerLoading } from 'components/ServiceContainers/ServiceContainerLoading';
+import { Toolbar } from 'components/Toolbar/Toolbar';
+import { ToolbarGroup } from 'components/Toolbar/ToolbarGroup';
+import { ToolbarItem } from 'components/Toolbar/ToolbarItem';
+import { TooltipWrapper } from 'components/TooltipWrapper/TooltipWrapper';
+
+const spaceItemsLg: FlexProps['spaceItems'] = { default: 'spaceItemsLg' };
+
+interface IAnalyzedArtifactsListProps {
+  serviceContainerArtifacts: IServiceContainerState<AnalyzedArtifactPage>;
+  columns?: string[];
+  componentId: string;
+}
+
+const defaultColumns = [
+  analyzedArtifactEntityAttributes['artifact.id'].id,
+  analyzedArtifactEntityAttributes['artifact.identifier'].id,
+  analyzedArtifactEntityAttributes['artifact.targetRepository.repositoryType'].id,
+  analyzedArtifactEntityAttributes['artifact.artifactQuality'].id,
+  analyzedArtifactEntityAttributes['artifact.buildCategory'].id,
+  analyzedArtifactEntityAttributes['artifact.filename'].id,
+  analyzedArtifactEntityAttributes['artifact.build'].id,
+  analyzedArtifactEntityAttributes.builtFromSource.id,
+  analyzedArtifactEntityAttributes.brewId.id,
+];
+
+/**
+ * Component displaying list of Analyzed Artifacts.
+ *
+ * @param serviceContainerArtifacts - Service Container for AnalyzedArtifacts
+ * @param componentId - Component ID
+ */
+export const AnalyzedArtifactsList = ({
+  serviceContainerArtifacts,
+  columns = defaultColumns,
+  componentId,
+}: IAnalyzedArtifactsListProps) => {
+  const sortOptions: ISortOptions = useMemo(
+    () =>
+      getSortOptions({
+        entityAttributes: analyzedArtifactEntityAttributes,
+        defaultSorting: {
+          attribute: analyzedArtifactEntityAttributes['artifact.identifier'].id,
+          direction: 'asc',
+        },
+        customColumns: columns,
+      }),
+    [columns]
+  );
+  const { getSortParams } = useSorting(sortOptions, componentId);
+
+  const { storageValue: isArtifactIdentifierParsed, storeToStorage: storeIsArtifactIdentifierParsed } = useStorage<boolean>({
+    storageKey: StorageKeys.isArtifactIdentifierParsed,
+    initialValue: false,
+  });
+
+  const [expandedArtifacts, setExpandedArtifacts] = useState<string[]>([]);
+  const [areAllArtifactsExpanded, setAreAllArtifactsExpanded] = useState<boolean | undefined>(undefined);
+
+  const setArtifactExpanded = (analyzedArtifact: AnalyzedArtifact, isExpanding = true) =>
+    setExpandedArtifacts((prev) => {
+      const others = prev.filter((id) => id !== analyzedArtifact.artifact!.identifier);
+      return isExpanding ? [...others, analyzedArtifact.artifact!.identifier] : others;
+    });
+  const isArtifactExpanded = (analyzedArtifact: AnalyzedArtifact) =>
+    expandedArtifacts.includes(analyzedArtifact.artifact!.identifier);
+
+  const expandOnFilterIds: string[] = [analyzedArtifactEntityAttributes.builtFromSource.id];
+
```

Comment:
`areBuildArtifactsExpanded` and `{columns.includes(artifactEntityAttributes.build.id) && <Th width={10} aria-label="Build Icon" />}` and other related things are missing, is it missing intentionally?

Reply:
added back

Reply:
Please could you keep exactly the same logic which is available on ArtifactList component? That one was well-tested and structure there was intentional.

Reply:
done